api/dating_app/src/main/java/com/psap/dating_app/model/User.java

```diff
@@ -93,28 +92,24 @@ public class User {
     @Column(name = "points", nullable = false)
     private Integer points;
 
-    @NotNull(message = "Block end is mandatory")
     @Column(name = "block_end", nullable = false)
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", timezone = "UTC")
     private Date blockEnd;
 
-    @NotBlank(message = "Blocked is mandatory")
+    @NotNull(message = "Blocked is mandatory")
     @Column(name = "blocked", nullable = false)
     private Boolean blocked;
 
-    @NotBlank(message = "Star sign is mandatory")
     @Column(name = "star_sign", nullable = false)
     private String starSign;
 
-    @NotBlank(message = "Personality type is mandatory")
     @Column(name = "personality_type", nullable = false)
     private String personalityType;
 
-    @NotBlank(message = "Love language is mandatory")

```

Comment:
Why delete these annotations?

Reply:
nes kol neturim to kazkokio testo kur sita nustato palikau kad blank gali but, kai implementuosim nustatyma sito padarysim not blank jei jis vps butinas bus